### ISSUE TITLE
Breakpoints list is global, not project-specific

### ISSUE THREAD

**Issue by sethladd** [dart-atom/dartlang#662]
When I'm sitting in project A, I see breakpoints for project B.

![screen shot 2015-12-19 at 10 30 30 am](https://cloud.githubusercontent.com/assets/5479/11914706/a33e1216-a63b-11e5-9653-5ebbfc4fafe5.png)
![screen shot 2015-12-19 at 10 30 25 am](https://cloud.githubusercontent.com/assets/5479/11914705/a33cf76e-a63b-11e5-8d88-8b464128b49f.png)

**Comment by devoncarew** [comment]
Yeah, it is global. You could have several projects open, some being packages in use by the running app. It's difficult to know which breakpoints will actually apply to an app. The breakpoints list will move out of the debugger area and into its own list. Hopefully this will make the ux less misleading.
